test/charms/test_integration/charmcraft.yaml

```diff
@@ -0,0 +1,43 @@
+---
+name: tracing-tester
+type: charm
+title: Tracing tester
+summary: Test tracing
+description: Test tracing
+
+base: ubuntu@24.04
+build-base: ubuntu@24.04
+platforms:
+    amd64:
+    arm64:
```

Comment:
Do we need to build for both platforms?

Reply:
Tempo charm is only published for amd64, so I guess it makes sense to match that in this tester charm, even if it's platform-agnostic.